forms-flow-api/src/formsflow_api/services/employeeDataService.py

```diff
@@ -0,0 +1,21 @@
+import json
+import requests
+from http import HTTPStatus
+
+class EmployeeDataService: 
+    
+    @staticmethod
+    def get_employee_data_from_bcgov(guid):
+      #TODO: get auth token from bcgov using @param guid
+      try:
+        responseFromBCGov = requests.get('https://analytics-testapi.psa.gov.bc.ca/apiserver/api.rsc/Datamart_Telework_employee_demo/',
```

Comment:
We can still add in the filter by GUID here. We can add a GUID to your keycloak user so you can test it out :)